Understanding Metrics for Public Cloud (#9006)
ovh/docs: pages/public_cloud/public_cloud_cross_functional/metrics_informations/guide.en-gb.md

---
title: 'Understanding Metrics in Public Cloud'
excerpt: 'Learn how Metrics work in OVHcloud Public Cloud, dashboards, subscriptions, and regional availability.'
updated: 2026-02-01
---

<style>
details>summary {
    color:rgb(33, 153, 232) !important;
    cursor: pointer;
}
details>summary::before {
    content:'\25B6';
    padding-right:1ch;
}
details[open]>summary::before {
    content:'\25BC';
}
</style>

## Objective

The objective of this document is to provide OVHcloud Public Cloud users with a comprehensive understanding of the Metrics feature, including:

> - Key Concepts – Explaining M4M (Dashboards) and M2C (Subscriptions) so users understand how to monitor and manage their resources.

Reviewer: ```suggestion
- Key Concepts – Explaining observability metrics dashboard and configuration to use them in Observability service so users understand how to monitor and manage their resources.
```

- Metrics Overview – Defining what Metrics are, how they work, and how they can help optimize cloud operations.
- Regional Roll-out – Highlighting the availability of Metrics across different regions, helping users plan and deploy workloads with full visibility.

By the end of this document, readers will be able to navigate the Metrics feature confidently, leverage dashboards and subscriptions effectively, and understand the service’s regional availability for better cost and resource management.

> ## Monitoring & Subscriptions: M4M and M2C

Reviewer: ```suggestion
## Observability metrics dashboard and configuration to use them in Observability service 
```

> M4M and M2C work together to provide a comprehensive experience: view metrics to understand the state of the cloud, and use them to take effective action. This approach ensures that users always have complete control over their resources and can optimize both performance and costs.

Reviewer: ```suggestion
Metrics dashboard and collect configuration with Observability service work together to provide a comprehensive experience: view metrics to understand the state of the cloud, and use them to take effective action. This approach ensures that users always have complete control over their resources and can optimize both performance and costs.
```

> ### M4M – Dashboards for Metrics Monitoring

Reviewer: ```suggestion
### Metrics Dashboards for Monitoring
```

> M4M represents the approach to visualizing and monitoring metrics. This feature allows users to:

Reviewer: ```suggestion
Metrics dashboard provide a visualization of metrics for a given resource for the purpose of monitoring its health and performance. This feature allows users to:
```

> - Centralize and visualize all metrics for their cloud services in real time and historically,

Reviewer: ```suggestion
- Centralize and visualize all metrics for a given cloud service in real time and historically,
```

> - Create customized dashboards to monitor performance, health, and resource usage,

Reviewer: ```suggestion
```

Reviewer: To remove We do not propose customer dashboard capabilities for now (in roadmap)

- Analyze trends and quickly detect anomalies.

The goal of M4M is to provide a comprehensive and intuitive dashboard, ensuring maximum visibility across the entire cloud infrastructure.

> ### M2C – Metrics Consumption and Subscriptions

Reviewer: ```suggestion
### Configuration to collect metrics in Observability service
```

> M2C corresponds to the management of Metrics consumption and use. It allows users to:

Reviewer: ```suggestion
Subscription is the activation of metrics collection in user's own tenant in observability service for advanced usage.
```

> - Subscribe to specific Metrics feeds and receive real-time notifications,

Reviewer: ```suggestion
- Subscribe to Prometheus metrics from a given resource to store them retention periode you defien (up to 13 months)
- Query your metrics in PromQL
```
